02.calendar/calendar.rb

```diff
@@ -0,0 +1,26 @@
+# !/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+def calendar
+  month = Date.today.month
+  year = Date.today.year
+
+  opt = OptionParser.new
+  opt.on('-m [VAL]') { |v| month = v.to_i }
+  opt.on('-y [VAL]') { |v| year = v.to_i }
+  opt.parse!(ARGV)
+
+  puts "       #{month}月 #{year}"
+  puts '日 月 火 水 木 金 土'
+  first_date = Date.new(year, month, 1)
+  last_date = Date.new(year, month, -1)
+  print '   ' * first_date.wday
+  (first_date..last_date).each do |date|
```

Comment:
```suggestion
  first_date.upto(last_date) do |date|
```

uptoを使うと少し短くできそうです
https://docs.ruby-lang.org/ja/latest/method/Date/i/upto.html